spring/jta/src/main/java/org/jboss/narayana/quickstart/spring/config/DummyXAConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat, Inc. and/or its affiliates, and individual
+ * contributors by the @authors tag. See the copyright.txt in the
+ * distribution for a full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.narayana.quickstart.spring.config;
+
+import com.arjuna.ats.jbossatx.jta.RecoveryManagerService;
+import org.jboss.narayana.quickstart.spring.xa.DummyXAResource;
+import org.jboss.narayana.quickstart.spring.xa.DummyXAResourceRecovery;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+@Configuration
+@Import(RecoveryManagerConfig.class)
+public class DummyXAConfig {
+    @Autowired
+    RecoveryManagerService rms;
+
+    @Bean
+    public DummyXAResource xaResource() {
+        rms.addXAResourceRecovery(xaResourceRecovery());
+        return new DummyXAResource();
```

Comment:
@tomjenkinson @gytis just point out that it register the resource helper when constructing the xa resouce bean